### DISCUSSION TITLE
/tr towncollect [resource_name]

### DISCUSSION THREAD

**Discussion by EcoSMP** [TownyAdvanced/TownyResources#96] [Ideas]
We run into an issue on our servers where towns/nations have too many resources to collect daily. The resources do drop onto the ground if there isn't enough space in the player's inventory but we get complaints very often about this, specifically from nation leaders who sometimes have much more than one inventory to collect. We would really like to have a small feature added to help with this.

Our idea is instead of just the normal /tr towncollect and /tr nationcollect commands, there could be /tr towncollect [resource_name] & /tr nationcollect [resource_name] so players can collect individual resources at a time giving them time to go store one resource before moving onto others. 

We would greatly appreciate you considering adding this for us! This is a complaint we get relatively often and we aren't quite sure what to do about it other than to ask you to add this small feature. Thank you again for the hard work you continue to put into the entire lineup of Towny plugins!

**Comment by LlmDl** [comment]
Did you want a preview copy of 0.2.3 to test out the new inventory system?

**Reply by EcoSMP** [reply]
That would be great, thank you for getting back to me!
